internal/renderer/helm.go

```diff
@@ -215,40 +215,49 @@ func makeChart(appname, targetDir string) error {
 	return ioutil.WriteFile(filepath.Join(targetDir, "Chart.yaml"), hmetadata, 0644)
 }
 
+func typeMeta(stackVersion string) metav1.TypeMeta {
+	return metav1.TypeMeta{
+		Kind:       "stacks.compose.docker.com",
+		APIVersion: stackVersion,
+	}
+}
+
+func objectMeta(appname string) metav1.ObjectMeta {
+	return metav1.ObjectMeta{
+		Name:      internal.AppNameFromDir(appname),
+		Namespace: "default", // FIXME
```

Comment:
I think it should be removed, as the "default" namespace is unlikely to be the user's default namespace.